src/components/Summary/SummaryDefineCompany.vue

```diff
@@ -67,7 +66,8 @@ import { FolioNumber, BusinessContactInfo, OfficeAddresses } from '@/components/
 import { EntityFilterMixin } from '@/mixins'
 
 // Enums
-import { EntityTypes } from '@/enums'
+import { CorpTypeCd, RouteNames } from '@/enums'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/index'
```

Comment:
You didn't want to re-export this in Enum Mixin (as is done in Filings UI)?

(This can be done later. Don't want to hold you up.)

Reply:
Oops, nope i missed this one. Thanks. will update!

Reply:
Yep, you're right. Was 2 others i missed! 😬